.github/workflows/release-tasks.yml

```diff
@@ -84,12 +84,21 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install dependencies
-        run: sudo apt-get install -y python3-setuptools
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y python3-setuptools python3-psutil clang cmake
+
+      - name: Build lit requirements
+        run: |
+          mkdir build && cd build
+          cmake ../llvm -DCMAKE_BUILD_TYPE=Release
+          cmake --build utils/FileCheck/ -j $(nproc)
+          cmake --build utils/not/ -j $(nproc)
 
       - name: Test lit
         run: |
           cd llvm/utils/lit
-          python3 lit.py tests
+          PATH=$(pwd)/../../../build/bin:$PATH python3 lit.py tests
```

Comment:
is this needed? I think if you invoke lit from the build dir it will pick up filecheck from the same dir.

Reply:
Considering RoboTux's suggestion, I'm closing this as resolved.